lib/bridge/IrcHandler.js

```diff
@@ -443,7 +443,10 @@ IrcHandler.prototype.onMetadata = Promise.coroutine(function*(req, client, msg)
         let response = yield this.ircBridge.getAppServiceBridge().getIntent().createRoom({
             createAsClient: false,
             options: {
-                name: "IRC Admin Room",
+                name: `${client.server.domain} Admin`,
```

Comment:
I like this. The only problem I see is that the domains may end up being ugly (think `ipv6.chat.freenode.net` or equiv). Should we instead prioritise the "description" field in the config or whatever it was? That would then be "Freenode".